### DISCUSSION TITLE
增加自定义命令排序功能

### DISCUSSION THREAD

**Discussion by sonichy** [linuxdeepin/developer-center#2860] [Features Request & Ideas | 特性请求 & 头脑风暴]
增加自定义命令排序功能，另外，左键执行命令，右键弹出删除菜单，删除菜单不要做到编辑对话框里。
![alt](https://i0.hdslb.com/bfs/album/92914e40118149e717935d624319b7723b6a8bea.png)

**Comment by ch593030323** [comment]
感谢您的建议，我们会合理考虑此功能
